sdl2promela/stop_condition/model.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from typing import List, Union
+
+
+class Expression:
+    """Stop Condition base class for all expression."""
+
+    pass
+
+
+class BinaryExpression(Expression):
+    """Stop Condition base class for binary expressions."""
+
+    lhs: Expression
+    """Left hand side of binary expression."""
+    rhs: Expression
+    """Right hand side of binary expression."""
+
+    def __init__(self, lhs: Expression, rhs: Expression):
+        super().__init__()
+        self.lhs = lhs
+        self.rhs = rhs
+
+
+class OrExpression(BinaryExpression):
```

Comment:
Is there are benefit to having one class per operator?

Reply:
This is my approach to modeling domain using OOP. I do not think that this is better or worse than other methods, this was my intuition.